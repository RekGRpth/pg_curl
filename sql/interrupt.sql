\unset ECHO
\set QUIET 1
\pset format unaligned
\pset tuples_only true
\pset pager off
BEGIN;
SET LOCAL client_min_messages = WARNING;
CREATE EXTENSION IF NOT EXISTS pg_curl;
END;
BEGIN;
select curl_easy_reset();
select curl_easy_setopt_url('https://httpbin.org/delay/2');
set statement_timeout = '1s';
select curl_easy_perform();
END;
