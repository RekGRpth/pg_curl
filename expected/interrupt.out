\unset ECHO
t
t
ERROR:  canceling statement due to statement timeout
